packages/fxa-auth-server/package.json

```diff
@@ -73,7 +76,7 @@
     "mozlog": "2.2.0",
     "mysql": "2.15.0",
     "mysql-patcher": "0.7.0",
-    "node-uap": "git+https://github.com/vladikoff/node-uap.git#9cdd16247",
+    "node-uap": "git+https://github.com/dannycoates/node-uap.git#96dc1f9f224422ec184395b6408cd1fc40ee452a",
```

Comment:
we need a node-uap with a pinned version of uap-core. unpinned causes some hard to debug test failures